website/next.config.mjs

```diff
@@ -20,13 +20,14 @@ const withNextra = nextra({
   },
   transform(content, { route }) {
     if (route) {
+      const routeSegments = route.split('/')
       return `
-import { getNavItems } from '@/navigation'
+import { getNavItems } from '${'../'.repeat(routeSegments.length - 1)}navigation'
```

Comment:
why did you remove `@` alias in tsconfig? seems it worked well?

Reply:
It didn't! Try running `pnpm lint` in your branch.

Reply:
I think ESlint config has a way to pick aliases 🤔

Reply:
<img width="580" alt="image" src="https://user-images.githubusercontent.com/7361780/225793885-c242718d-f792-423b-8921-5c4014757b61.png">

Reply:
Oh yeah Prettier fails which prevents `tsc` from even running... try `pnpm lint:fix`

Reply:
We should probably replace the `&&`s with `;` so each command runs regardless of the result of the previous one.

Reply:
to fix you can replace the below in `package.json`

```diff
-"typecheck": "tsc --noEmit",
+"typecheck": "pnpm --filter @graphprotocol/docs typecheck",
```

Reply:
Ohhh, thank you @B2o5T! Done in https://github.com/graphprotocol/docs/pull/319/commits/a0717fa035aa6a0ed4fc4524bf737a0e270db4d9, and then I reverted removing the alias in https://github.com/graphprotocol/docs/pull/319/commits/e8baaf649f0933da1b572dfba25ce2fe0aab29fe.

By the way, there are still 4 ESLint warnings:

<img width="774" alt="CleanShot 2023-03-16 at 22 20 47@2x" src="https://user-images.githubusercontent.com/1059139/225795731-88c9cbed-0ee2-4f45-a8db-ebddf92b6728.png">

Those are weird, I don't really understand why TypeScript thinks they are `any`.

Reply:
I will take a look tomorrow!